app/routes/demos/12-mini-game/styles.module.css

```
@@ -0,0 +1,12 @@
+/* Fredoka – loaded only on mini-game route */
+@font-face {
+  font-family: "Fredoka";
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Fredoka" (font-family-name-quotes)

(font-family-name-quotes)